theano/tensor/tests/test_opt.py

```diff
@@ -1977,8 +1981,8 @@ def test4(self):
         y = tensor.vector('y')
         f = function([x, y], tensor.exp(x + y)[0], mode=mode_opt)
 
-        # Check stacktrace was copied over correctly after opt was applied
-        self.assertTrue(check_stack_trace(f, ops_to_check='all'))
+        # opt doesn't apply, so no need for check_stack_trace
+        #self.assertTrue(check_stack_trace(f, ops_to_check='all'))
```

Comment:
The comment is correct.
You should add a space after the `#` so that pep8 is respected.